### ISSUE TITLE
Edge "Help Text" causing issue

### ISSUE THREAD

**Issue by jjleavitt** [kevinchappell/formBuilder#560]
### Description:
  Edge returns error complaining "Object doesn't support property or method 'append'" when help text is defined and loaded into builder or render engine.
  
  proposed and tested on my environment fix is to change layout.js line 26 from:

```
        if (help) {
          label.append(help);
        }
```

to

```
        if (help) {
          label.appendChild(help);
        }
```
Works both chrome and ie

### Environment Details:

 - formBuilder Version:  2.5.3
 - Browser:  Edge
 - OS:  Win 10

### Expected Behavior
  form to work
### Actual Behavior
  form breaks
### Steps to Reproduce
  drop on any field
  add help text
  save
  load in builder or render
### Screenshot - *(optional)*
![image](https://user-images.githubusercontent.com/491053/28291701-73907406-6b00-11e7-85ea-d0817215d665.png)

**Comment by jordan314** [comment]
+1
Thanks for this fix, if you need to edit in form-render.min.js change `n&&e.append(n);` to `n&&e.appendChild(n);`

**Comment by kevinchappell** [comment]
resolved in #596
